aea/cli/core.py

```diff
@@ -170,3 +171,4 @@ def add_key(ctx: Context, type_, file):
 cli.add_command(push)
 cli.add_command(publish)
 cli.add_command(fetch)
+cli.add_command(generate)
```

Comment:
@5A11 please sort all the `add_command` statements alphabetically by their argument. I know it was not your code but makes the code more readable for the next developer :)

Reply:
Done